setup.py

```diff
@@ -362,6 +363,7 @@ def run(self):
     + extras["docs_specific"]
     + extras["sklearn"]
     + extras["modelcreation"]
+    + extras["accelerate"]
```

Comment:
Can we put it in `"all"` instead? It should be there and then will be automatically included in dev.